src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CSharp.Analyzers/HtmlHelperAnalyzer.cs

```diff
@@ -41,14 +36,11 @@ public override void Initialize(AnalysisContext context)
 
         private static readonly DiagnosticDescriptor Rule = new(DiagnosticId, Title, MessageFormat, Category, DiagnosticSeverity.Warning, isEnabledByDefault: true, description: Description);
 
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(Rule); } }
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         private void AnalyzeIdentifiers(SyntaxNodeAnalysisContext context)
         {
-            if (context.Node is not IdentifierNameSyntax identifier)
-            {
-                return;
-            }
+            var identifier = (IdentifierNameSyntax)context.Node;
```

Comment:
The cast will always succeed because RegisterSyntaxNodeAction is passed SyntaxKind.IdentifierName